main/src/resources/doc/index.js

```diff
@@ -0,0 +1,31 @@
+const storeKey = "flix-html-docs:use-dark-theme";
+
+const body = document.querySelector("body");
+body.classList.remove("no-script");
+
+function setTheme(dark) {
+    if (dark) {
+        body.classList.remove("light");
+        body.classList.add("dark");
+    } else {
+        body.classList.remove("dark");
+        body.classList.add("light");
+    }
+}
+
+let dark;
```

Comment:
You have both a local variable called dark and you use it as a parameter to setTheme.

Reply:
Obviously that "works" but it is not pretty :)